docs/projects/trenchboot-aem.md

```diff
@@ -0,0 +1,301 @@
+---
+template: giscus.html
+---
+
+# Trenchboot as Anti Evil Maid
+
+## Abstract
+
+The firmware is the heart of the security of a given system and should be
+always up-to-date to maintain the security of the computer. However, being up
+to date does not prevent the firmware vulnerabilities to appear. The Static
+Root of Trust (SRT) like Unified Extensible Firmware Interface (UEFI) Secure
+Boot and measured boot provided by the firmware is not always sufficient to
+establish a secure environment for an operating system. In case the firmware is
+compromised, it could inject malicious software into operating system
+components and prevent the machine owner from detecting it. Silicon vendors
+implement alternative technologies to establish a Dynamic Root of Trust (DRT)
+to provide a secure environment for operating system launch and integrity
+measurements. These integrity measurements either from SRT or DRT can be used
+for operating system attestation. However, DRT technologies are designed in a
+way to provide the ability to establish a secure environment for integrity
+measurements at any arbitrary point of time instead of relying on the firmware
+which requires machine reset to establish the aforementioned secure
+environment.
+
+The usage of DRT technologies like Intel Trusted Execution Technology (TXT) or
+AMD Secure Startup becomes more and more significant, for example, Dynamic Root
+of Trust for Measurement (DRTM) requirements of Microsoft Secured Core PCs
```

Comment:
I think we should [labels](https://spec.commonmark.org/0.30//#link-reference-definitions) or just link, why we keep things in so weird way? Current format cannot be transferred easily to MoU.

```suggestion
of Trust for Measurement (DRTM) requirements of [Microsoft Secured Core PCs](https://some.link)
```

Reply:
Done for all links using labels